src/System.Globalization.Calendars/tests/System.Globalization.Calendars.Tests.csproj

```diff
@@ -0,0 +1,118 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{9574CEEC-5554-411B-B44C-6CA9EC1CEB08}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <AssemblyName>System.Globalization.Calendars.Tests</AssemblyName>
+    <RootNamespace>System.Globalization.Calendars.Tests</RootNamespace>
+  </PropertyGroup>
```

Comment:
I suspect for now you're going to need to add ```<UnsupportedPlatforms>Linux;OSX</UnsupportedPlatforms>``` here, as we've not yet implemented calendar support on Unix, and without that, these tests will fail in CI.